charts/datadog-operator/values.yaml

```diff
@@ -58,6 +58,8 @@ logLevel: "info"
 maximumGoroutines:
 # supportExtendedDaemonset -- If true, supports using ExtendedDaemonSet CRD
 supportExtendedDaemonset: "false"
+# forwardOperatorMetrics -- If false, disables sending operator metrics to Datadog
+forwardOperatorMetrics: "true"
```

Comment:
for ease of finding it, could we use the same name as the flag? (i imagine you named it this way to have an active verb and match the other parameter, which i agree is unfortunate)

also my two cents on the description: i prefer something more explicit, but since there's precedence for this format as well i'll leave it up to you!